src/components/core/Flex/Flex.js

```diff
@@ -1,28 +1,20 @@
 import styled from 'styled-components';
-import {
-	flexWrap,
-	flexDirection,
-	alignItems,
-	justifyContent,
-} from 'styled-system';
 import Box from '../Box';
+import propTypes from '@styled-system/prop-types';
 
-const Flex = styled(Box)(
-	{
-		display: 'flex',
-	},
-	flexWrap,
-	flexDirection,
-	alignItems,
-	justifyContent
-);
+export const Flex = styled(Box)({});
+
+Flex.defaultProps = {
+	display: 'flex',
+	justifyContent: 'center',
```

Comment:
I'd not set such defaults since it can confuse users. To me this is on the same level as margins etc.

Reply:
Yeah, I'm not so sure about this too. I've copied that from somewhere like Rebass Grid I think ;-)